pkg/clock/clock_test.go

```diff
@@ -0,0 +1,23 @@
+package clock_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openmeterio/openmeter/internal/testutils"
+	"github.com/openmeterio/openmeter/pkg/clock"
+)
+
+func TestClock(t *testing.T) {
+	clock.SetTime(testutils.GetRFC3339Time(t, "2024-06-30T15:39:00Z"))
+	defer clock.ResetTime()
+
+	now := clock.Now()
+	diff := now.Sub(testutils.GetRFC3339Time(t, "2024-06-30T15:39:00Z"))
+	if diff < 0 {
+		diff = -diff
+	}
+	assert.True(t, diff < time.Second)
```

Comment:
i'm not sure i understand the point of this test

Reply:
This test just ensures that the calculation is pseudo correct. If I would have messed up the sign of the drift this would fail.